packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/from/V1_MappingTransformer.ts

```diff
@@ -430,7 +430,9 @@ const transformPurePropertyMapping = (
     element.property,
     false,
   );
-  purePropertyMapping.source = ''; // @MARKER: GRAMMAR ROUNDTRIP --- omit this information during protocol transformation as it can be interpreted while building the graph
+  purePropertyMapping.source = transformPropertyMappingSource(
```

Comment:
Had to change this to fix engine roundtrip tests

Reply:
We should not do this. This needs to get fixed in the backend. The backend should not produce this property as the compiler can resolve it. One thing about the grammar tests is that when it fails it is not always on studion to fix. This is why you can skip steps on the test.

Reply:
BTW lets make sure we are not setting the property.class value when localMappingProperty is defined. IN the transformer is where we want to do this logic.

Reply:
will skip the tests then